src/registry_schemas/schemas/receiver.json

```diff
@@ -0,0 +1,61 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/annual_report",
+    "required": [
+        "receiver"
+    ],
+    "type": "object",
+    "title": "Appoint Receiver / Receiver Manager Filing",
+    "properties": {
+        "receiver": {
```

Comment:
Receiver Manager should be a new party type

https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/party.json#L69